api/src/websocket/services/socket-event-dispatcher.service.ts

```
@@ -73,6 +73,14 @@ export class SocketEventDispatcherService implements OnModuleInit {
       }
 
       const [_, handler] = foundHandler;
+
+      await new Promise<void>(async (resolve, reject) => {
+        req.session.reload((_err) => {
+          if (_err) reject();
+          resolve();
+        });
+      });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)